CONTRIBUTING.md

```diff
@@ -104,7 +104,7 @@ In Jina we use git's pre-commit hooks in order to make sure the code matches our
 It's easy to configure it:
 
 1. `pip install pre-commit`
-1. `pre-commit install .`
+1. `pre-commit install`
```

Comment:
Oh I didn't notice this PR, and I made the same correction in an already merged PR :P